src/frontend/qt_sdl/AudioSettingsDialog.cpp

```diff
@@ -17,6 +17,13 @@
 */
 
 #include <stdio.h>
+#include <QtGlobal>
+#if QT_VERSION >= 0x060000
```

Comment:
Use the `QT_VERSION_CHECK()` macro instead.
https://doc.qt.io/qt-6/qtglobal.html#QT_VERSION_CHECK